src/main/java/core/basesyntax/serviceimpl/WriterServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.WriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeToFile(String filePath,List<String> lines) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            writer.write("fruit,quantity");
```

Comment:
Consider extracting the header 'fruit,quantity' into a constant field. This would make it easier to manage and change if needed in the future, and it will help to avoid hardcoding values directly in the service methods.

Reply:
agree, call it a REPORT_HEADER